tsfresh/feature_extraction/settings.py

```diff
@@ -146,7 +146,9 @@ def __init__(self):
             "energy_ratio_by_chunks": [{"num_segments": 10, "segment_focus": i} for i in range(10)],
             "ratio_beyond_r_sigma": [{"r": x} for x in [0.5, 1, 1.5, 2, 2.5, 3, 5, 6, 7, 10]],
             "linear_trend_timewise": [{"attr": "pvalue"}, {"attr": "rvalue"}, {"attr": "intercept"},
-                                      {"attr": "slope"}, {"attr": "stderr"}]
+                                      {"attr": "slope"}, {"attr": "stderr"}],
+            "count_above": [{"t": t} for t in [-1, 0, 1]],
```

Comment:
I don't think we should give defaults for this feature, as the usefullness highly depends on the value range for your time series.

Reply:
Indeed, so you are suggesting I should remove them from the ComprehensiveFCParameters?

Reply:
Yes, I would remove it

Reply:
By removing the features from the ComprehensiveFCParameters, it throws an error during the tests execution. The _test_default_calculates_all_features_ test fails, as it checks if, by default, the ComprehensiveFCParameters object contains all the existing features.

Reply:
If any, then I would go only with the 0 as default parameter

Reply:
I added the 0 as the default parameter